### PR TITLE
[Feature](torc) Support insert only transactional hive table on be side.

### PR DESCRIPTION
**Pull request by kaka11chen**
# Proposed changes

Issue Number: close #xxx

## Problem summary

Describe your changes.

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by kaka11chen** [comment]
run buildall

File: be/src/vec/exec/format/orc/vorc_reader.cpp [hunk 1]
```diff
@@ -284,6 +301,44 @@ Status OrcReader::_init_read_columns() {
...
+        orc_cols.emplace_back(type.getFieldName(i));
+        orc_cols_lower_case.emplace_back(_get_field_name_lower_case(&type, i));
+        const orc::Type* sub_type = type.getSubtype(i);
+        if (sub_type->getKind() == orc::TypeKind::STRUCT) {
+            _init_orc_cols(*sub_type, orc_cols, orc_cols_lower_case);
+        }
+    }
+}
+
+bool OrcReader::_check_acid_schema(const orc::Type& type) {
```
Comment (morningman) [inline be/src/vec/exec/format/orc/vorc_reader.cpp:316]: How about let FE tell BE it is a torc or not? So that you don't need to check this for every file?

Comment (kaka11chen) [reply be/src/vec/exec/format/orc/vorc_reader.cpp:316]: Good idea. It will be improved in the full acid implementation.

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**